testgrid/cmd/configurator/main.go

```diff
@@ -74,6 +79,9 @@ func gatherOptions() (options, error) {
 	if o.validateConfigFile && o.output != "" {
 		return o, errors.New("--validate-config-file doesn't write the proto anywhere")
 	}
+	if (o.prowConfig == "") != (o.prowJobConfig == "") {
+		return o, errors.New("--prow-config and --prow-job-config must be specified together")
```

Comment:
Note this on the flag help messages as well?

Reply:
Done.